Packs/Zendesk/Integrations/Zendeskv2/README.md

```diff
@@ -26,13 +24,18 @@ If you are upgrading from a previous of this integration, see [Breaking Changes]
     | Use system proxy settings |  | False |
     | Trust any certificate (not secure) |  | False |
 
-4. Click **Test** to validate the URLs, token, and connection.
+4. Click **Test** to validate the URLs, token, and connection. 
+    (The test does not ensure sufficient permissions for all integration commands.)
```

Comment:
Don't we want to test permissions for all the roles and notify the user what fails rather than let him guess ?

Reply:
He does not need to guess I added documentation for each command - what permissions it requires.